gix-sec/src/identity.rs

```diff
@@ -123,6 +123,11 @@ mod impl_ {
                 );
 
                 if result != ERROR_SUCCESS {
+                    // We cannot determine ownership, so we default to reduced trust (false) rather
+                    // than failing completely.
+                    if result == ERROR_INVALID_FUNCTION {
+                        return Ok(false);
+                    }
```

Comment:
It seems to me that this describes the workaround more accurately and clearly, as we do still return errors for conditions other than the unavailability of (or inability to retrieve) security information.

```suggestion
                if result != ERROR_SUCCESS {
                    if result == ERROR_INVALID_FUNCTION {
                        // We cannot obtain security information, so we default to reduced trust
                        // (false) rather than failing completely.
                        return Ok(false);
                    }
```